release-process.rst

```diff
@@ -64,65 +79,158 @@ Example time line with only one major version at a time
+	- Release: ``$rd + 42`` (Aug 15, 2024)
+
+- Beta 2: ``$rd + 56`` (Aug 29, 2024)
+- Beta 3: ``$rd + 70`` (Sep 12, 2024)
+
+At feature freeze:
+
+- All features requiring an RFC must have passed by the voting mechanism, and
+  SHOULD be merged prior to feature freeze.
+
+After feature freeze, with blessing of the release managers:
+
+- Merging RFC requiring features is still allowed.
+- Features that do not require an RFC are still allowed.
+- Optimisations and internal ABI and API changes are also still allowed.
+
+Release Candidates
+------------------
+
+.. list-table::
+   :header-rows: 0
+   :stub-columns: 1
+
+   * - Release Candidate 1
+     - * Tag: ``$rd + 82`` (Sep 24, 2024)
+       * Release: ``$rd + 84`` (Sep 26, 2024)
+   * - Release Candidate 2
+     - ``$rd + 98`` (Oct 10, 2024)
+   * - Release Candidate 3
+     - ``$rd + 112`` (Oct 24, 2024)
+   * - Release Candidate 4
+     - ``$rd + 126`` (Nov 07, 2024)
+
+More release candidates MAY be added on a two-week cycle, if necessary.
+
+With the first release candidate:
+
+- Internal API numbers MUST be updated (``PHP_API_VERSION``,
+  ``ZEND_MODULE_API_NO``, and  ``ZEND_EXTENSION_API_NO``).
+- The release branch (``PHP-8.4``) MUST be created.
+
+After the first release candidate:
+
+- There MUST NOT be any API and ABI changes in subsequent RCs.
+- There MUST NOT be any new features, small or otherwise, in subsequent RCs.
+
+
+General Availability
+--------------------
+
+.. list-table::
+   :header-rows: 0
+   :stub-columns: 1
+
+   * - x.y.0 (8.4.0)
+     - * Tag: ``$rd + 138`` (Nov 19, 2024)
+       * Release: ``$rd + 140`` (Nov 21, 2024)
+
+The GA release MUST be released from the last Release Candidate tag (RC4 or
+later). There MUST NOT be any changes between the last Release Candidate tag and
+the GA tag (with exception to files such as `NEWS` and other files where the
+PHP version number must change for the GA release).
+
+Bug Fix and Security Releases
+-----------------------------
+
+After the general availability release:
 
-Timeline example for a release
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Until the end of year 2 (e.g., for PHP 8.4: until Dec 31, 2026):
 
-- June
+	- A new release every 4 weeks, synchronised with other release branches.
+	- Bug fixes and security fixes.
 
-  - Decisions which features or changes will be in the next release
-  - 1st release alpha (may have many alpha)
+- Until the end of year 3 (e.g., for PHP 8.4: until Dec 31, 2027):
 
-- At least one release per month, more at wish
-- September, RC phases, biweekly release
+	- Security fixes, and fixes to address regressions introduced during a
+	  normal bug fix release.
+	- Updates to ABI incompatible versions of dependent libraries on Windows.
+	- Release only when there is a security issue or regression issue to
+	  address.
+	- Release when an otherwise normal bug fix release for other branches is
+	  also made. Exceptions can be made for high risk security issues or high
+	  profile regressions.
```

Comment:
This comment got "outdated" and hidden because of another update, so adding it again...

Does this statement contradict the previous statement to "Release only when there is a security issue or regression issue to address?"

I'm not entirely sure what this statement means. Does this mean, for the 3rd year, the N.N.x version will always have a release when any of the other release branches has a bug fix release? Even if there isn't a security patch? If so, I don't think I understood that from the RFC.

Reply:
I meant to say that a security bug fix release should be on the same date as the (normal) bug fix releases of the other branches. Like we already currently do.

Reply:
Got it. How about this change for clarity?

```suggestion
	- Security fix and regression releases SHOULD occur on the same date as bug
	  fix releases for the other branches. Exceptions can be made for high risk
	  security issues or high profile regressions.
```